_pages/committees/organization.md

```diff
@@ -21,3 +21,8 @@ sidebar:
 
 <a href="https://apepa.github.io/">Pepa Atanasova</a>, University of Copenhagen<br/>
 <a href="https://www.linkedin.com/in/julius-cheng-0040799/">Julius Cheng</a>, University of Cambridge
+
+### Publicity Chairs
+
+<a href="mailto:tetreaul@gmail.com">Joel Tetreault</a>, Dataminr<br/>
```

Comment:
This is a bit inconsistent, we used URLs for everyone above, if you feel like it, we could ask Isabelle for the websites in addition 😃

Reply:
Replaced with URLs!